client/guidestours/style.scss

```diff
@@ -58,3 +84,47 @@ $zoom-scale: 5; // the multiplier determining the size of the animated rings
 .guidestours__pointer__ring:after {
 	animation-delay: #{ $animation-speed / 4 };
 }
+
+.guidestours__buttonrow__choice {
+	button {
+		width: 48%;
+	}
+	button:nth-child(1) {
+		margin-right: 4%;
+	}
+	button.guidestours__button__secondary {
+		background: $gray-light;
```

Comment:
I'm asking because we used to have gray buttons but we chose to remove them in favor of white ones. If we are considering bringing them back it should be at the component level.

cc @mattmiklic @MichaelArestad for past discussions around this.